### ISSUE TITLE
BUG: "index.d.ts is not a module" in 0.18.4 when grapesjs is imported with "import * as grapesjs from 'grapesjs';"

### ISSUE THREAD

**Issue by varadero** [artf/grapesjs#4240]
### GrapesJS version

- [X] I confirm to use the latest version of GrapesJS

### What browser are you using?

Chrome 99

### Reproducible demo link

No link

### Describe the bug

**How to reproduce the bug?**
1. I was using grapesjs 0.18.2 in my typescript angular class imported like this - `import * as grapesjs from 'grapesjs';`
2. Upgraded to 0.18.4

**What is the expected behavior?**
No errors

**What is the current behavior?**
Typescript complains at the line `import * as grapesjs from 'grapesjs';` with the following error:
```
node_modules/grapesjs/index.d.ts' is not a module.ts(2306)
```
The `'grapesjs'` part of the import has red squiggles. I suppose this is because `index.d.ts` was introduced in 0.18.4 (https://github.com/artf/grapesjs/releases/tag/v0.18.4)

![image](https://user-images.githubusercontent.com/7610713/161810283-0517cb32-c7c6-4cd9-9230-955ccaa18e00.png)

Version 0.18.3 does not have this problem.

If is necessary to execute some code in order to reproduce the bug, paste it here below:
```ts
import * as grapesjs from 'grapesjs';
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by artf** [comment]
Should be fixed here https://github.com/artf/grapesjs/pull/4216 
I guess you should disable TS for the current version.

**Comment by ghoshpushpendu** [comment]
> Should be fixed here #4216 I guess you should disable TS for the current version.

Disabling TypeScript solving the issue.

**Comment by aharishsundhar** [comment]
@artf im using grapesjs latest version ^0.18.4 
i install npm grapesjs to run it its error on index.d.ts file can you please update issue
npm install:
![Screenshot from 2022-06-03 12-12-01](https://user-images.githubusercontent.com/72383148/171802669-b8bde687-a7c4-4a9d-8489-e00d1d3ca165.png)
manually change it local:
 
![Screenshot from 2022-06-03 12-15-15](https://user-images.githubusercontent.com/72383148/171802774-65a17fc7-033c-4d07-85d5-bb2999c629cf.png)
manully update it my side can you please update the modules side 
I REPLACE TYPE ANY
![Screenshot from 2022-06-03 12-15-28](https://user-images.githubusercontent.com/72383148/171802903-665e96a8-3941-4482-9ef2-d47b0a1efabc.png)
